.vitest/extend.ts

```diff
@@ -1,6 +1,6 @@
 import { expect } from 'vitest';
 
-const ITERATIONS = 100;
+const ITERATIONS = 2500;
```

Comment:
This will have to be rebased into the `toReasonablySatisfy` method.

Reply:
Oh woops, I raised this to test some things out 😅 
I'll rebase later and revert it to 100

Reply:
@THEtheChad I changed it to a environment variable